conda_build/environ.py

```diff
@@ -345,7 +345,11 @@ def system_vars(env_dict, prefix):
     if 'MAKEFLAGS' in os.environ:
         d['MAKEFLAGS'] = os.environ['MAKEFLAGS']
 
-    d['CPU_COUNT'] = get_cpu_count()
+    if 'CPU_COUNT' in os.environ:
+        d['CPU_COUNT'] = os.environ['CPU_COUNT']
```

Comment:
> make -j${CPU_COUNT}

This would indeed be an adjustment for conda-forge. Though this is exactly what many of us had done with our recipes previously.